src/main/webapp/app/pages/usageAnalysisPage/ResourceUsageDetailsTable.tsx

```diff
@@ -37,23 +37,28 @@ export default class ResourceUsageDetailsTable extends React.Component<
     this.timeTypeToggleValue = value;
   }
 
+  Filters = () => {
+    return (
+      <Row>
+        <UsageToggleGroup
+          defaultValue={this.timeTypeToggleValue}
+          toggleValues={[
+            ToggleValue.RESULTS_IN_TOTAL,
+            ToggleValue.RESULTS_BY_MONTH,
+          ]}
+          handleToggle={this.handleTimeTypeToggleChange}
+        />
+      </Row>
+    );
+  };
```

Comment:
Can we move this to a standalone functional component?

Reply:
@zhx828 I don't think we can move it outside of the class component as it needs to be able to modify the `timeTypeToggleValue` which is a property of the class component and changes the table. Is this what you meant for me to do?